metrics/prometheus/prometheus.yml

```
@@ -12,14 +12,15 @@ alerting:
     static_configs:
     - targets: []
 scrape_configs:
-- job_name: prometheus
+- job_name: trin
   honor_timestamps: true
   scrape_interval: 15s
   scrape_timeout: 10s
   metrics_path: /metrics
   scheme: http
   follow_redirects: true
   enable_http2: true
+  fallback_scrape_protocol: PrometheusText0.0.4
   static_configs:
   - targets:
-    - host.docker.internal:9100
+    - localhost:9100
```

```suggestion
    - localhost:9100
    - host.docker.internal:9100
```

please add both so the dashboard works on both linux and macos